tools/distpartitioning/data_shuffle.py

```diff
@@ -287,8 +304,11 @@ def exchange_node_features(rank, world_size, node_feature_tids, ntype_gnid_map,
     logging.info(f'[Rank: {rank}] Total time for node feature exchange: {timedelta(seconds = end - start)}')
     return own_node_features, own_global_nids
 
-def exchange_graph_data(rank, world_size, node_features, node_feat_tids, edge_data,
-        id_lookup, ntypes_ntypeid_map, ntypes_gnid_range_map, ntid_ntype_map, schema_map):
+def exchange_graph_data(rank, world_size, node_features, edge_features, 
+        node_feat_tids, edge_feat_tids, 
+        edge_data, id_lookup, ntypes_ntypeid_map, 
+        ntypes_gnid_range_map, etypes_geid_range_map, 
```

Comment:
update the docstring of parameters accordingly?

Reply:
updated all the relevant doc strings.